developer_guides/testbench/build_testbench.rst

```diff
@@ -0,0 +1,110 @@
+.. _build_testbench:
+
+Build Testbench and do first run
+################################
+
+This assume the firmware git repository has been cloned and a shell is
```

Comment:
this guide assumes

Reply:
and we should state that the assumption is based on the installation instructions and include the link.

Reply:
Done!